src/common/emit-and-fire/index.js

```diff
@@ -7,8 +7,14 @@
 function emitAndFire(widget, eventName, eventArg) {
     const originalEmit = widget.emit;
     let event;
+    //if window.CustomEvent evals true on ie11 but does not support new CustomEvent
```

Comment:
I know it's small, but could you move this comment above the try statement? It seems more relevant the closer it is to the code you wrote.